website/docs/components/form/super-select/partials/guidelines/guidelines.md

```diff
@@ -61,10 +77,78 @@ The footer defaults to hidden, but when enabled shows a counter (selected count
 
 ![Custom content enabled before the list in both single and multiple variants](/assets/components/form/super-select/super-select-single-variant-footer-with-counter.png =626x*)
 
-## Content
+##### Multiple variant
+The afterOptions and footer default to visible. This is the preferred pattern where users are able to expedite the deselection process.
+
+![Custom content enabled before the list in both single and multiple variants](/assets/components/form/super-select/super-select-multiple-variant-footer-without-counter.png =757x*)
+
+Show selected will change the list to display only selected list items while toggling the label of that button to “Show all."
+
+![Flow showing the behavior of the show selected button and how it toggles the list to only display selected list items](/assets/components/form/super-select/super-select-flow-select-all.png =836x*)
+
+Additionally, “clear selected” will wipe out the selections and revert the SuperSelect to its defined default state.
+
+![Flow showing the behavior of the clear selected button and how it reverts the SuperSelect's state to the default one](/assets/components/form/super-select/super-select-flow-clear-selected.png =836x*)
+
+##### Single and multiple variant footer with custom content
+In either of the two variants, you are able to fill the footer with custom content. This is helpful when wanting to mimic the footer but still customize its contents.
+
+![Showing the footer with custom content inside.](/assets/components/form/super-select/super-select-single-multiple-footer-custom-content.png =836x*) 
+
+#### CustomContent
+If enabled, yielded content will occupy this space.
+
+![Showing the afterOptions slot with custom content](/assets/components/form/super-select/super-select-after-options-custom-content.png =836x*)
 
-- {description}
-- {description}
-- {description}
-- {description}
-- {description}
\ No newline at end of file
+## List
+
+### Positioning
+Lists can be positioned to the left, center or right of the trigger, and above or below the trigger to fit more appropriately within the UI. This defaults to the bottom left with collision detection.
```

Comment:
@KristinLBradley are we doing center as well? I looked at our dropdown list and it seems as though that doesn't have a center position (only left and right). Let me know so I can update this if we are not including center as well.

Reply:
I had added an example after I initially pushed the examples showing left & right alignment and I see Alex has added additional examples.

Here's for the Single variant:
<img width="1092" alt="image" src="https://github.com/hashicorp/design-system/assets/108769823/f518f93f-439f-492c-856c-f0ad27ee356a">

Reply:
Thanks alex said the same, keeping in docs, thanks for clarity.